scrapping/test.ipynb

```diff
@@ -292,6 +292,13 @@
     "    \n",
     "\n"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```

Comment:
I don't know why the jupyter notebook used to scrap added itself a cell and how I pushed it. However it does not affect anything